converter.go

```diff
@@ -1098,6 +1108,26 @@ func arrowToRecord(ctx context.Context, record arrow.Record, pool memory.Allocat
 					}
 				}
 
+				newCol = tb.NewArray()
+				defer newCol.Release()
+			}
+		case textType:
+			if arrowBatchesUtf8ValidationEnabled(ctx) && col.DataType().ID() == arrow.STRING {
+				tb := array.NewStringBuilder(pool)
+				defer tb.Release()
+
+				for i := 0; i < int(numRows); i++ {
+					if col.(*array.String).IsValid(i) {
+						stringValue := col.(*array.String).Value(i)
+						if !utf8.ValidString(stringValue) {
+							logger.Error("Invalid utf8 detected!")
```

Comment:
Can we add some more context here? If I saw such message in logs without any more info I would be completely confused. How about `Invalid UTF-8 characters detected while reading query response` maybe include column name (if we have it)? WDYT?

Reply:
Previously I log out the whole cell content but I think that's too risky, we should not log any customer data.

I guess column name is ok?

Reply:
@sfc-gh-pfus , done.

Reply:
Yes, logging value is too risky. Column name is good enough. Thank you!